testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -1048,8 +1048,8 @@ impl RelayerThread {
                 &mut op_signer,
                 1,
             )
-            .ok_or_else(|| {
-                warn!("Failed to submit block-commit bitcoin transaction");
+            .map_err(|e| {
+                warn!("Failed to submit block-commit bitcoin transaction: {}", e);
                 NakamotoNodeError::BurnchainSubmissionFailed
```

Comment:
If TransactionSubmissionFailure is capturing the internal error, I wonder if BurnchainSubmissionFailed should as well?

Reply:
Addressed in https://github.com/stacks-network/stacks-core/commit/ca885d07e101fd7bbe8db0c4b6ca62cc23dc1033